File: tyrant/fsharp/delete/tyrant_delete.fs
// -------------------------------------------------------------------
//
//	tyrant_delete.fs
//
//					Aug/06/2011
//
// -------------------------------------------------------------------
open System
open System.Text
open Microsoft.FSharp.Core
open System.IO

open Memcached.ClientLibrary
open mcached_manipulate
// -------------------------------------------------------------------
[<EntryPoint>]
let main (args : string[] ) =

  printfn "*** 開始 ***"
  let key = args.[0]

  printfn "%s" key

  let serverlist = [|"127.0.0.1:1978" |]

  let pool = SockIOPool.GetInstance()

  pool.SetServers(serverlist)
  pool.Initialize()

  let mc = new MemcachedClient()

//  if (mc.KeyExists(key)) then
  let _ = mc.Delete (key)

  let keys = [|"t4761";"t4762";"t4763";"t4764";"t4765";"t4766";"t4767";"t4768";"t4769";|]
  display_proc (mc,keys)
//
  pool.Shutdown()
  printfn "*** 終了 ***"
  0
// -------------------------------------------------------------------

File: tyrant/csharp/delete/tyrant_delete.cs
// -------------------------------------------------------------
//
//	tyrant_delete.cs
//
//						Jun/23/2011
// -------------------------------------------------------------
using System;

using Memcached.ClientLibrary;


// -------------------------------------------------------------
class mcached_update
{

// -------------------------------------------------------------
static void Main (string[] args)
{
	Console.WriteLine ("*** 開始 ***");

	String	key = args[0];

	Console.WriteLine (key);

	String[] serverlist = { "172.20.208.233:1978" };

	SockIOPool pool = SockIOPool.GetInstance();
	pool.SetServers(serverlist);
	pool.Initialize();

	MemcachedClient mc = new MemcachedClient();


	if (mc.KeyExists(key))
		{
		mc.Delete (key);
		}

	pool.Shutdown();

	Console.WriteLine ("*** 終了 ***");
}
// -------------------------------------------------------------
}
// -------------------------------------------------------------
